### PR TITLE
Upgrade version to 3.16.0, and API version to 2021-04-10

### PR DESCRIPTION
**Pull request by blueww**
Thanks for contribution! Please go through following checklist before sending PR.

### PR Branch Destination

- For Azurite V3, please send PR to `main` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Add Change Log

Add change log for the code change in `Upcoming Release` section in `ChangeLog.md`.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -4,15 +4,28 @@
...
+- Fixed issue that startCopyFromURL and copyFromURL API not respect `--disableProductStyleUrl` parameter in parse source Uri.
+
 Queue:
 
 - Fixed issue that queue list result is not in alphabetical order.
 
-## 2021.10 Version 3.15.0
+Table:
+
+- Fixed issues with deleting entities using empty string for RowKey.
```
Comment (XiaoningLiu) [inline ChangeLog.md:24]: Why moving Table section from 3.15.0 to 3.16.0?

**Review by XiaoningLiu — APPROVED** [review]
Overall looks good but change to ChangeLog.md may need further update

File: ChangeLog.md (hunk 1 above)
Comment (blueww) [reply ChangeLog.md:24]: The 2 table changes are added after 3.15.0 is out.
They are from PR https://github.com/Azure/Azurite/pull/1235 and https://github.com/Azure/Azurite/pull/1279
the 2 PRs added the changelog to release version 3.15.0. But not in 3.15.0.
So I move the 2 change log to 2.16.0

**Pull request merged**